Jenkinsfile

```diff
@@ -33,9 +33,29 @@ node {
 
     if(env.BRANCH_NAME == "master") {
       stage('Publish Gem') {
-        sshagent(['govuk-ci-ssh-key']) {
-          govuk.runRakeTask("publish_gem --trace")
+        def version = sh(
+          script: /ruby -e "puts eval(File.read('govuk_navigation_helpers.gemspec'), TOPLEVEL_BINDING).version.to_s"/,
+          returnStdout: true
+        ).trim()
+
+        echo 'This version: ' + version
+
+        def shouldPush = sh(
```

Comment:
Makes sense to me - does that mean we're going to try and go ahead with this approach?